packages/main/src/StepInput.hbs

```diff
@@ -0,0 +1,68 @@
+<div
+	id="{{_id}}"
+	class="ui5-step-input-root"
+	style="{{styles}}"
+	@keydown="{{_onkeydown}}"
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+>
+	<div
+		class="ui5-step-icon ui5-step-dec"
+		title="{{decIconTitle}}"
+		?interactive="{{_decIconInteractive}}"
+	>
+		<ui5-icon
+			id="{{_id}}-dec"
+			name="{{decIconName}}"
+			tabindex="-1"
+			accessible-name="{{decIconTitle}}"
+			?interactive="{{_decIconInteractive}}"
+			?defocused="{{_disableFocus}}"
+			@click="{{_decValue}}"
+			@focusout="{{_onButtonFocusOut}}"
+			input-icon
+		></ui5-icon>
+	</div>
+
+	<!-- INPUT -->
+	<ui5-input
+			id="{{_id}}-inner"
+			class="ui5-step-input-input"
+			placeholder="{{_placeholder}}"
+			type="{{type}}"
+			value="{{_valuePrecisioned}}"
+			?disabled="{{disabled}}"
+			?required="{{required}}"
+			?readonly="{{readonly}}"
+			value-state="{{valueState}}"
+			data-sap-focus-ref
+			text-align="{{textAlign}}"

```

Comment:
The ui5-input does not have text-align property?

Reply:
Done, removed